src/PointwiseFunctions/Elasticity/ConstitutiveRelations/CubicCrystal.cpp

```diff
@@ -0,0 +1,80 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "PointwiseFunctions/Elasticity/ConstitutiveRelations/CubicCrystal.hpp"
+
+#include <array>
+#include <pup.h>  // IWYU pragma: keep
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"  // IWYU pragma: keep
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "ErrorHandling/Assert.hpp"
+#include "Utilities/MakeWithValue.hpp"
+
+// IWYU pragma: no_forward_declare Tensor
+
+namespace Elasticity::ConstitutiveRelations {
+
+CubicCrystal::CubicCrystal(double c_11, double c_12, double c_44) noexcept
+    : c_11_(c_11), c_12_(c_12), c_44_(c_44) {
+  ASSERT(c_11_ >= c_12_,
+         "c_11 must be smaller than c_12. This is because the youngs_modulus "
```

Comment:
the error message doesn't agree with the assert condition.  also print the values of c_11 and c_12 in the message